### ISSUE TITLE
Odd skin naming

### ISSUE THREAD

**Issue by Barrykilby** [Laravel-Backpack/CRUD#1393]
# Bug report

"dick-light"

Just noticed this one, bit silly but I would imagine it will create some confusion over what tone your colour scheme ends up with. 

https://github.com/Laravel-Backpack/CRUD/tree/2ee1227a40ed89230764751db7dad521e431df3b/src/public/tinymce/skins

Also interestingly part of the dick crud commit, I presume this is a type or a language thing?
EDIT: IOC its related to Richard, I guess my point about what colour you end up with still stands though?!

### What I did:
Noticed it

### What I expected to happen:
"dark-light" I would guess judging from the colours in the skin

### What happened:
Some sniggering

### What I've already tried to fix it:
I make it my policy to leave dicks alone

**Comment by tabacitu** [comment]
Hi @Barrykilby ,

Yup. That name was actually intentional :-) It's not an easter egg or something, we just seemed to have missed one spot when changing names.

Backpack is actually the 4th admin panel framework that I've created. The 3rd was called Dick, just for laughs. We enjoyed writing ```git pull dick```, ```git push dick```. Also, I hosted it on usedick.com, where we used to joke that "_Dick is the only tool you'll ever need_" and stuff like that. Fun times :-) Dick became quite popular, actually (turns out people like Dick), but at the next rewrite I decided to change the name from Dick to Backpack. I had a lot of requests for this - professionals were skeptical in using Dick because they were afraid their clients might see the name somewhere. So yeah... we had to give up Dick. It wasn't easy, but I think we're better for it :-)

You're totally right though, this old name shouldn't be here, on this filename. But there's already an issue suggesting we create a different color scheme, so this will definitely get fixed then ;-)

Thanks, cheers!
